sql/mob_pool_mods.sql

```diff
@@ -429,7 +434,7 @@ INSERT INTO `mob_pool_mods` VALUES (3824,29,2,1); -- ASSIST: 2
 INSERT INTO `mob_pool_mods` VALUES (3853,28,23,1); -- EXP_BONUS: 23
 
 -- Tiamat
-INSERT INTO `mob_pool_mods` VALUES (3916,29,50,0);   -- MDEF: 50
+INSERT INTO `mob_pool_mods` VALUES (3916,29,130,0);  -- MDEF: 100
```

Comment:
You set 130 but wrong MDEF: 100? Not sure which ones its supposed to be

Reply:
Set back to original value